<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectiveDateWizardPage</class>
 <widget class="QWizardPage" name="EffectiveDateWizardPage">
  <attribute name="title">
    <string>Effective date</string>
  </attribute>
  <layout class="QVBoxLayout">
    <property name="margin">
      <number>11</number>
    </property>
    <property name="spacing">
      <number>6</number>
    </property>
    <item>
      <widget class="QLabel" name="m_effectiveDateLabel">
        <property name="text">
          <string>1</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <widget class="QLabel" name="m_effectiveDateNoteLabel">
        <property name="text">
          <string>1</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer68_3">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>30</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <widget class="QLabel" name="TextLabel1_7">
            <property name="text">
              <string>When should the changes become active?</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="KMyMoneyDateInput" name="m_effectiveChangeDateEdit"/>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer69_2">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>87</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
